src/app/dataset-block/metadata-block/components/event-details/components/set-transform-event/set-transform-event.component.html

```diff
@@ -0,0 +1,59 @@
+
+<div class="d-flex justify-content-end">
```

Comment:
Let's try reusing the HTML template between `SetPollingSource` and `SetTransform` events. I don't think there will be any difference, you can just point to the same HTML file from multiple components.

Reply:
Made one template for all events